assembl/static2/js/app/components/administration/resourcesCenter/index.jsx

```diff
@@ -0,0 +1,119 @@
+        load={(fetchPolicy: FetchPolicy) => load(client, fetchPolicy)}
+        loading={loading}
+        postLoadFormat={postLoadFormat}
+        createMutationsPromises={createMutationsPromises(client, lang)}
+        save={save}
+        validate={validate}
+        mutators={{
+          ...arrayMutators
+        }}
+        render={({ handleSubmit, pristine, submitting }) => (
+          <div className="admin-content">
+            <AdminForm handleSubmit={handleSubmit} pristine={pristine} submitting={submitting}>
+              <div className="form-container">
+                <Field
+                  editLocale={editLocale}
+                  name="pageTitle"
+                  component={MultilingualTextFieldAdapter}
+                  label={I18n.t('administration.resourcesCenter.pageTitleLabel')}
+                  required
+                />
+                <Field
+                  name="pageHeader"
+                  component={FileUploaderFieldAdapter}
+                  label={I18n.t('administration.resourcesCenter.headerImageLabel')}
+                />
+                <div className="separator" />
+              </div>
+
+              <FieldArrayWithActions
+                name="resources"
+                renderFields={({ name }) => (
+                  <React.Fragment>
+                    <Field
+                      editLocale={editLocale}
+                      name={`${name}.title`}
+                      component={MultilingualTextFieldAdapter}
+                      label={`${I18n.t('administration.resourcesCenter.titleLabel')} ${editLocale.toUpperCase()}`}
+                      required
+                    />
+                    <Field
+                      key={`${name}-text-${editLocale}`}
+                      editLocale={editLocale}
+                      name={`${name}.text`}
+                      component={MultilingualRichTextFieldAdapter}
+                      label={`${I18n.t('administration.resourcesCenter.textLabel')} ${editLocale.toUpperCase()}`}
+                    />
+                    <Field
+                      componentClass="textarea"
+                      name={`${name}.embedCode`}
+                      component={TextFieldAdapter}
+                      label={I18n.t('administration.resourcesCenter.embedCodeLabel')}
+                    />
+                    <Field
+                      name={`${name}.img`}
+                      component={FileUploaderFieldAdapter}
+                      label={I18n.t('administration.resourcesCenter.imageLabel')}
+                    />
+                    <Field
+                      name={`${name}.doc`}
+                      component={FileUploaderFieldAdapter}
+                      label={I18n.t('administration.resourcesCenter.documentLabel')}
+                    />
+                  </React.Fragment>
+                )}
+                titleMsgId="administration.resourcesCenter.editResourceFormTitle"
```

Comment:
I think the jsx is easier to read if you put the renderFields prop as the last prop of the component

Reply:
you're right, thanks